### ISSUE TITLE
Streaming Issue

### ISSUE THREAD

**Issue by asimclasio** [alexgolec/tda-api#25]
Hello, I am trying to start the streaming but getting the following error. 
I can get the history with same credentials but streaming is not working. 

```
import asyncio
import json
import tda
import config

from selenium import webdriver

token_path = config.token_path
api_key = config.api_key
redirect_uri = config.redirect_uri
primary_account_id = config.account_id

c = tda.auth.easy_client(api_key, redirect_uri, token_path,
        webdriver_func=lambda: webdriver.Chrome())

client = tda.streaming.StreamClient(c, account_id=config.account_id)

async def read_stream():
    await client.login()
    await client.quality_of_service(client.QOSLevel.EXPRESS)

    await client.nasdaq_book_subs(['GOOG'])
    client.add_nasdaq_book_handler(
            lambda msg: print(json.dumps(msg, indent=4)))

    while True:
        await client.handle_message()

asyncio.get_event_loop().run_until_complete(read_stream())

```

```
_________________________________________________________________________________________
Traceback (most recent call last):
  File "C:/Users/owner/PycharmProjects/tda/streamtda.py", line 29, in <module>
    asyncio.get_event_loop().run_until_complete(read_stream())
  File "C:\Users\owner\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:/Users/owner/PycharmProjects/tda/streamtda.py", line 19, in read_stream
    await client.login()
  File "C:\Users\owner\PycharmProjects\tda\venv\lib\site-packages\tda\streaming.py", line 325, in login
    await self._await_response(request_id, 'ADMIN', 'LOGIN')
  File "C:\Users\owner\PycharmProjects\tda\venv\lib\site-packages\tda\streaming.py", line 194, in _await_response
    resp = await self._receive()
  File "C:\Users\owner\PycharmProjects\tda\venv\lib\site-packages\tda\streaming.py", line 117, in _receive
    raw = await self._socket.recv()
  File "C:\Users\owner\PycharmProjects\tda\venv\lib\site-packages\websockets\protocol.py", line 509, in recv
    await self.ensure_open()
  File "C:\Users\owner\PycharmProjects\tda\venv\lib\site-packages\websockets\protocol.py", line 812, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: code = 1000 (OK), no reason

Process finished with exit code 1
```

**Comment by songkevinc** [comment]
I'm having the same issue. Any luck finding a solution?

**Comment by asimclasio** [comment]
No Still the same.I am still waiting for Alex for further guidance.

**Comment by alexgolec** [comment]
Given that this works perfectly for so many others, perhaps you need to sign some agreements and/or enable streaming? Do you have streaming in your subscriptions? Have you signed exchange agreements?

**Comment by asimclasio** [comment]
Hi Alex, just re-confirmed, have a streaming subscription, and exchange
agreement is also signed. But if it's working for other I am closing the
issue. Thanks.

**Comment by alexgolec** [comment]
That's fair. Also, feel free to hop on to our discord if you want to discuss this further: discord.gg/nfrd9gh

**Comment by mLewisLogic** [comment]
Just to chime in on this:
I had this exact same issue. Very frustrated by it.
Turns out the problem was that I was trying to use a "linked" account (not the login's primary account). Once I switched logins to the login that owned the account I wanted to trade the problem went away.

Sort of a silly setup, but lets me manage HSA and brokerage from the same login, but seems problematic on TDA's end for streaming.
